app/components/chart_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if title %>
+  <h4 class="chart-title"><%= title %></h4>
+<% end %>
+
+<% if subtitle %>
+  <h5 class="chart-subtitle"><%= subtitle %></h5>
+<% end %>
+
+<% if header %>
+  <p><%= header %></p>
```

Comment:
Could the header/footer just render whatever partial its been provided rather than assuming the content is wrapped in a paragraph? E.g. we might want a couple of paras, or some bullet points in a footer.